tests/test_document.py

```diff
@@ -230,8 +230,10 @@ def test_doc_splitter_headings(self, reset_db) -> None:
         for md_str in [','.join(t) for t in md_tuples]:
             print(f'{md_str=}')
             chunks_t = pxt.create_view(
-                'chunks', doc_t,
-                iterator=DocumentSplitter.create(document=doc_t.doc, separators='sentence', metadata=md_str))
+                            'chunks', doc_t,
```

Comment:
The formatting is still off. Did you run pylint?

See elsewhere in the code base for examples of multi-line function calls. Indentation is generally 4 spaces.